v3/src/components/case-table/collection-table.tsx

```diff
@@ -243,51 +245,55 @@ export const CollectionTable = observer(function CollectionTable(props: IProps)
     }
   }
 
-const scrollInterval = useRef<NodeJS.Timeout | null>(null)
-const mouseY = useRef<number>(0)
+  const scrollInterval = useRef<NodeJS.Timeout | null>(null)
+  const mouseY = useRef<number>(0)
+
+  const marqueeSelectCases = useCallback((startIdx: number, endIdx: number) => {
+        const newSelectedRows = []
+        const start = Math.min(startIdx, endIdx)
+        const end = Math.max(startIdx, endIdx)
+        for (let i = start; i <= end; i++) {
+          newSelectedRows.push(i)
+        }
+        const selectedCaseIds = newSelectedRows
+                                  .map(idx => collectionCaseIdFromIndex(idx, data, collectionId))
+                                  .filter((id): id is string => id !== undefined)
+        setOrExtendSelection(selectedCaseIds, data)
```

Comment:
I was wrong before about using `selectCases`. But you should use `setSelectedCases` here. `setOrExtendSelection` is intended to be used when you sometimes want to set the selection, and sometimes want to modify it. Here, you're always setting the selection.